app/code/Magento/Customer/Controller/Address.php

```diff
@@ -29,6 +32,11 @@
      */
     protected $_addressRepository;
 
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    protected $customerRepository;
```

Comment:
Please, make this property private

Reply:
this property is protected couse it is used in `Magento\Customer\Controller\Address\FormPost` and other which extends this base controller

Reply:
We recommend using composition instead of inheritance according to our [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html), that's why this property should be private. Please, make it private and define the new property in every child if you need it there.